### PR TITLE
NO-ISSUE: Capture output of VM installation process

### PR DESCRIPTION
**Pull request by pmtk**

### CODE REVIEW

File: test/bin/scenario.sh [hunk 1]
```diff
@@ -197,22 +197,36 @@ launch_vm() {
...
+
+    set +e
+    sudo virt-install \
          --noautoconsole \
          --name "${full_vmname}" \
          --vcpus 2 \
          --memory 4092 \
          --disk "pool=${VM_STORAGE_POOL},size=30" \
          --network network=default,model=virtio \
-         --events on_reboot=restart \
```
Comment (ggiguash) [inline test/bin/scenario.sh:207]: I think we need to leave this event, or attempts to reboot VMs would result in their shutdown. 
We have a few tests that reboot VMs, right?

File: test/bin/scenario.sh [hunk 2]
```diff
@@ -197,22 +197,36 @@ launch_vm() {
...
-         --extra-args "inst.ks=${kickstart_url}" \
-         --wait; then
-        record_junit "${vmname}" "launch_vm" "OK"
+         --extra-args "inst.ks=${kickstart_url} console=ttyS0" \
+         --console "file,path=${install_log}" \
+         --wait "${wait}"
+    local -r install_result=$?
+    set -e    
+    sudo chown "$(whoami):$(whoami)" "${install_log}"
+    mv "${install_log}" "${install_log_dest}"
```
Comment (ggiguash) [inline test/bin/scenario.sh:221]: How does this work if the `install_log_dest` is `${WEB_SERVER_URL}`

File: test/bin/scenario.sh [hunk 3]
```diff
@@ -197,22 +197,36 @@ launch_vm() {
     # FIXME: variable for vcpus?
     # FIXME: variable for memory?
     # FIXME: variable for ISO
-    if timeout "${VM_BOOT_TIMEOUT}" sudo virt-install \
+
+    local -r wait=$((VM_BOOT_TIMEOUT / 60))
+    local -r install_log=$(mktemp)
+    local -r install_log_dest="${WEB_SERVER_URL}/scenario-info/${SCENARIO}/vms/${vmname}/vm_install.log"
```
Comment (ggiguash) [inline test/bin/scenario.sh:203]: WEB_SERVER_URL?

Comment (pmtk) [reply test/bin/scenario.sh:203]: Ah, right. I didn't check it, thought it's local fs...

File: test/bin/scenario.sh (hunk 2 above)
Comment (pmtk) [reply test/bin/scenario.sh:221]: I added this bit without testing :)

File: test/bin/scenario.sh [hunk 4]
```diff
@@ -197,22 +197,38 @@ launch_vm() {
...
          --location "${VM_DISK_DIR}/${boot_blueprint}.iso" \
-         --extra-args "inst.ks=${kickstart_url}" \
-         --wait; then
-        record_junit "${vmname}" "launch_vm" "OK"
+         --extra-args "inst.ks=${kickstart_url} console=ttyS0" \
+         --console "file,path=${install_log}" \
+         --wait "${wait}"
+    local -r install_result=$?
+    set -e
+    sudo chown "$(whoami):$(whoami)" "${install_log}"
```
Comment (ggiguash) [inline test/bin/scenario.sh:222]:
```suggestion
    sudo chown "$(whoami)". "${install_log}"
```

**Comment by pmtk** [comment]
/hold

`--autoconsole` needs tty, when I have more time I'll see if ssh with `-t` will help or it's no possible in CI env at all

```
Creating domain...                                          |         00:00     
error: Cannot run interactive console without a controlling TTY

Running text console command: virsh --connect qemu:///system console rhel-9.2-microshift-source-backup-prune-host1

Domain is still running. Installation may be in progress.
Waiting 15 minutes for the installation to complete.
```

**Pull request closed without merging**